src/platforms/x86_64_linux_musl.rs

```diff
@@ -213,15 +214,7 @@ fn build_crt(
                 "-DCMAKE_C_COMPILER='clang'",
                 "-DCMAKE_CXX_COMPILER='clang++'",
                 "-DLLVM_ENABLE_PROJECTS='compiler-rt'",
-                format!(
-                    "-DLLVM_TARGETS_TO_BUILD='{}'",
-                    targets
-                        .into_iter()
-                        .map(|platform| platform.to_string())
-                        .collect::<Vec<String>>()
-                        .join(";")
-                )
-                .as_str(),
+                "-DLLVM_TARGETS_TO_BUILD='X86'",
```

Comment:
We have a constant with `to_string()` defined in the `Platform` enum.

Reply:
Awesome, updated these lines.